src/core/namespace.c

```diff
@@ -3967,6 +3970,201 @@ static const char* const private_bpf_table[_PRIVATE_BPF_MAX] = {
 
 DEFINE_STRING_TABLE_LOOKUP(private_bpf, PrivateBPF);
 
+#include "bpf-delegate-configs.inc"
+
+DEFINE_STRING_TABLE_LOOKUP(bpf_delegate_cmd, uint64_t);
+DEFINE_STRING_TABLE_LOOKUP(bpf_delegate_map_type, uint64_t);
+DEFINE_STRING_TABLE_LOOKUP(bpf_delegate_prog_type, uint64_t);
+DEFINE_STRING_TABLE_LOOKUP(bpf_delegate_attach_type, uint64_t);
+
+char* bpf_delegate_commands_to_string(uint64_t u) {
```

Comment:
Maybe better to make this slightly generic, and take a function pointer to bpf_delegate_cmd_to_string() and friends. That greatly reduce the number of lines.